.gitlab/ci/sdk-nightly.yml

```diff
@@ -144,10 +158,73 @@ demisto-sdk-nightly:run-commands-against-instance:
     - section_end "Run Lint"
     - *change-file-ids
     - *upload-entities-to-cortex-xsoar
-    - section_start "Destroy instances"
-    - python3 ./Tests/scripts/destroy_instances.py $ARTIFACTS_FOLDER $ARTIFACTS_FOLDER/env_results.json "$INSTANCE_ROLE" "$TIME_TO_LIVE"
-    - section_end "Destroy instances"
+    # - section_start "Destroy instances"
+    # - python3 ./Tests/scripts/destroy_instances.py $ARTIFACTS_FOLDER $ARTIFACTS_FOLDER/env_results.json "$INSTANCE_ROLE" "$TIME_TO_LIVE"
+    # - section_end "Destroy instances"
 
+demisto-sdk-nightly:run-commands-against-instance-e2e:
+  tags:
+    - gke
+  extends:
+    - .default-job-settings
+    - .sdk-nightly-schedule-rule-always
+  variables:
+    INSTANCE_ROLE: "Server Master"
+  stage: run-instances
+  script:
+    - !reference [.open-ssh-tunnel]
+    - python3 ./Tests/scripts/wait_until_server_ready.py "$INSTANCE_ROLE"
+    - !reference [.download-demisto-conf]
+    - |
+      export DEMISTO_API_KEY=$(cat $SECRET_CONF_PATH | jq -r '.temp_apikey')
+      export DEMISTO_BASE_URL="https://localhost:$(cat $ARTIFACTS_FOLDER/env_results.json | jq -r '.[0].TunnelPort')"
+      echo "Server URL: $DEMISTO_BASE_URL"
+    - section_start "uploading Files to ARTIFACTS_FOLDER"
+    - cp -Rf ./TestsData/DemistoSDKE2E /Packs
+    - section_end "end copying Files to ARTIFICATES_FOLDER"
+    - section_start "Upload Entities to XSOAR Instance" --collapsed
+    - *upload-entities-to-cortex-xsoar-e2e
+    - section_end "Upload Entities to XSOAR Instance"
+    - section_start "Download Entities from XSOAR Instance" --collapsed
+    # - demisto-sdk download -o Packs/DemistoSDKE2E/ -i DemistoSDKE2E --insecure -f
+    # - demisto-sdk download -o Packs/DemistoSDKE2E/ -i HelloWorld-Test --insecure -f
+    # - demisto-sdk download -o Packs/DemistoSDKE2E/ -i "Hello World IncidentField Test" --insecure -f
+    # - demisto-sdk download -o Packs/DemistoSDKE2E/ -i "Hello World Alert Test" --insecure -f
+    # - demisto-sdk download -o Packs/DemistoSDKE2E/ -i "Hello World Test - Incoming Mapper" --insecure -f
+    # - section_end "Download Entities from XSOAR Instance"
+    # - section_start "Format Content Items" --collapsed
+    # - demisto-sdk format -i Packs/HelloWorld/ -nv -ud -y
+    # - section_end "Format Content Items"
+    # - section_start "Run Validate" --collapsed
+    # - demisto-sdk validate -i Packs/HelloWorld/
+    # - section_end "Run Validate"
+    # - section_start "Run Lint" --collapsed
+    # - demisto-sdk lint -i Packs/HelloWorld/
+    # - section_end "Run Lint"
+    # - *change-file-ids
+    # - *upload-entities-to-cortex-xsoar
+    # - section_start "Destroy instances"
+    # - python3 ./Tests/scripts/destroy_instances.py $ARTIFACTS_FOLDER $ARTIFACTS_FOLDER/env_results.json "$INSTANCE_ROLE" "$TIME_TO_LIVE"
+    # - section_end "Destroy instances"
```

Comment:
commented out on purpose?

Reply:
yes

Reply:
why? is it intended to stay commented out when merging?

Reply:
No No